runtime/strings/strings.cpp

```diff
@@ -17,8 +17,6 @@
 
 extern "C" {
 
-#define KCHAR char
```

Comment:
Nice!

Reply:
There's more of this to be removed but I'm going to do it incrementally